packages/dds/tree/src/simple-tree/simpleSchema.ts

```diff
@@ -162,6 +162,23 @@ export type SimpleNodeSchema =
 	| SimpleObjectNodeSchema
 	| SimpleRecordNodeSchema;
 
+/**
+ * Information about allowed types.
+ *
+ * @alpha
+ * @sealed
+ */
+export interface SimpleAllowedTypes {
+	/**
+	 * True if this schema is included as a {@link SchemaStaticsAlpha.staged | staged} schema upgrade,
+	 * allowing the view schema be compatible with stored schema with (post upgrade) or without it (pre-upgrade).
+	 * New documents and schema upgrades will omit any staged schema.
+	 *
+	 * Undefined if derived from a stored schema.
```

Comment:
The fact that this differs between stored and view schema gives me some pause. I wonder if (at least longer term), we want slightly different variants of simple schema for stored vs view schema? What other differences are going to creep in over time here?

But for now this is probably fine. It is an alpha API. Maybe worth a quick `@privateRemarks` note about future considerations?